release-linux-deb-gcc/Dockerfile

```diff
@@ -0,0 +1,69 @@
+ARG DISTRO=debian
+ARG DISTRO_TAG=stretch
+FROM ${DISTRO}:${DISTRO_TAG}
+
+ENV DEBIAN_FRONTEND=noninteractive
+
+# Blocks in this file are created as such that we have as many cache hits as
+# possible between different Dockerfiles
+
+# Dependencies needed to compile OpenTTD
+RUN apt-get update && apt-get upgrade -y && apt-get dist-upgrade -y && \
+    apt-get install -y --no-install-recommends \
+    \
+    bzip2 \
+    git \
+    libc6-dev \
+    libfile-fcntllock-perl \
+    libfontconfig-dev \
+    libicu-dev \
+    liblzma-dev \
+    liblzo2-dev \
+    libsdl1.2-dev \
+    libxdg-basedir-dev \
+    make \
+    mercurial \
+    subversion \
+    tar \
+    xz-utils \
+    zlib1g-dev \
+    && rm -rf /var/lib/apt/lists/*
```

Comment:
remove cache directories as well?

Reply:
Docker has some nasty hooks that already do that for you :)